README.md

```diff
@@ -67,3 +67,20 @@ cAdvisor aims to improve the resource usage and performance characteristics of r
 ## Community
 
 Contributions, questions, and comments are all welcomed and encouraged! cAdvisor developers hang out on [Slack](https://kubernetes.slack.com) in the #sig-node channel (get an invitation [here](http://slack.kubernetes.io/)). We also have [discuss.kubernetes.io](https://discuss.kubernetes.io/).
+
+Please reach out and get involved in the project, we're actively looking for more contributers to bring on board!
+
+### Core Team
+* [@bobbypage, Google](http://github.com/bobbypage)
+* [@iwankgb, Independent](http://github.com/iwankgb)
+* [@creatone, Intel](http://github.com/creatone)
+* [@dims, VMWare](http://github.com/dims)
+* [@mrunalp, RedHat](http://github.com/mrunalp)
+
+### Frequent Collaborators
+* [@pehunt, RedHat](http://github.com/pehunt)
```

Comment:
nit, my username is haircommander :slightly_smiling_face:

Reply:
oops, fixed thanks!